backend/tests/conftest.py

```diff
@@ -117,3 +118,11 @@ def new_msg():
         "content": "Lorem Ipsum cos tam cos tam"
     }
     return msg
+
+
+@pytest.fixture
+def add_participants(app, _db):
+    db = _db
```

Comment:
Do not see a lot of reason for such assignment. Would say I'm fine with overwriting that symbol (just as `def add_participants(app, db)` or calling it at the end as `_db.session.commit()`.